domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/StopTimeArrivalTimeAfterDepartureTimeNotice.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.notice.error;
+
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.NoticeExporter;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.ErrorNotice;
+
+import java.io.IOException;
+
+public class StopTimeArrivalTimeAfterDepartureTimeNotice extends ErrorNotice {
+    public StopTimeArrivalTimeAfterDepartureTimeNotice(final String filename,
+                                                       final String arrivalTimeAsString,
+                                                       final String departureTimeAsString,
+                                                       final String tripId,
+                                                       final String stopSequence) {
```

Comment:
I will make the change but I think we should avoid Stringifying non String data at this level so that consumers of the report can leverage native JSON types.